features/fields_extensie.feature

```diff
@@ -11,57 +11,116 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
   - opschortingBijhouding reden en datum worden altijd meegegeven wanneer deze een waarde hebben.
 
   Achtergrond:
-    Gegeven: persoon met burgerservicenummer 999994086 heeft geboortedatum (01.03.10) in onderzoek
-    En persoon met burgerservicenummer 999993483 heeft geheimhoudingPersoonsgegevens=true
-    En persoon met burgerservicenummer 999992077 is overleden
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999994086 |
+    En de persoon heeft de volgende geboorte gegevens
+    | naam                                      | waarde   |
+    | datum (01.03.10)                          | 19860401 |
+    | land.code                                 | 6030     |
+    | plaats.code                               | 0518     |
+    | inOnderzoek.aanduidingGegevensInOnderzoek | TODO     |
+    | inOnderzoek.datum                         | 20200601 |
+    En het systeem heeft een persoon met de volgende gegevens
+    | naam                          | waarde    |
+    | burgerservicenummer           | 999993483 |
+    | geheimhoudingPersoonsgegevens | 7         |
+    En de persoon heeft de volgende geboorte gegevens
+    | naam        | waarde |
+    | plaats.code | 0518   |
+    En het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992077 |
+    En de persoon heeft de volgende geboorte gegevens
+    | naam        | waarde |
+    | plaats.code | 0518   |
+    En de persoon heeft de volgende overlijden gegevens
+    | naam | waarde |
+    | TODO | TODO   |
+    En de persoon heeft de volgende opschortingBijhouding gegevens
+    | naam       | waarde     |
+    | reden.code | overlijden |
+    | datum      | 20151001   |
+    En de waardetabel 'Gemeenten' heeft de volgende waarden
+    | code | omschrijving  |
+    | 0518 | 's-Gravenhage |
+    En de waardetabel 'Landen' heeft de volgende waarden
+    | code | omschrijving |
+    | 6030 | Nederland    |
 
```

Comment:
ik denk dat het de leesbaarheid vergroot wanneer ook deze feature wordt gegroepeerd in rules. Dus niet eerst in de algemene tekst de hele functionaliteit beschrijven en dan daaronder alle scenario's, maar per rule de bijbehorende voorbeelden.
Als we toch aan het aanpassen zijn, kan dat ook mooi mee...